From stable-bounces@linux.kernel.org Thu Dec  8 22:27:06 2005
Message-ID: <43991E34.60503@gmail.com>
Date: Fri, 09 Dec 2005 01:03:32 -0500
From: Michael Krufky <mkrufky@gmail.com>
To: stable@kernel.org
Cc: Mauro Carvalho Chehab <mchehab@brturbo.com.br>,
        lkml <linux-kernel@vger.kernel.org>, mkrufky@gmail.com
Subject: V4L/DVB: Fix analog NTSC for Thomson DTT 761X hybrid tuner

[PATCH] V4L/DVB: Fix analog NTSC for Thomson DTT 761X hybrid tuner

- Enable tda9887 on the following cx88 boards:
  pcHDTV 3000
  FusionHDTV3 Gold-T
- This ensures that analog NTSC video will function properly, without
  this patch, the tuner may appear to be broken.

Signed-off-by: Michael Krufky <mkrufky@m1k.net>
Signed-off-by: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/media/video/cx88/cx88-cards.c |    2 ++
 1 file changed, 2 insertions(+)

--- linux-2.6.14.3.orig/drivers/media/video/cx88/cx88-cards.c
+++ linux-2.6.14.3/drivers/media/video/cx88/cx88-cards.c
@@ -567,6 +567,7 @@ struct cx88_board cx88_boards[] = {
 		.radio_type     = UNSET,
 		.tuner_addr	= ADDR_UNSET,
 		.radio_addr	= ADDR_UNSET,
+		.tda9887_conf   = TDA9887_PRESENT,
 		.input          = {{
 			.type   = CX88_VMUX_TELEVISION,
 			.vmux   = 0,
@@ -711,6 +712,7 @@ struct cx88_board cx88_boards[] = {
 		.radio_type     = UNSET,
 		.tuner_addr	= ADDR_UNSET,
 		.radio_addr	= ADDR_UNSET,
+		.tda9887_conf   = TDA9887_PRESENT,
 		.input          = {{
                         .type   = CX88_VMUX_TELEVISION,
                         .vmux   = 0,
